style.css

```diff
@@ -0,0 +1,624 @@
+
+@import url('https://fonts.googleapis.com/css2?family=Kumbh+Sans&display=swap');
+
+
+
+div,section{
+    display: block;
+}
+
+body{
+    font-size: 1rem;
+    font-family: arial,sans-serif;
+    color: #222;
+    padding: 0;
+    margin: 0;
+}
+
+.headGoogle{
+    width: 50%;
+    margin: 0 auto;
+    font-family: 'Kumbh Sans', sans-serif;
+    text-align: center;
+    font-size: 2.3rem;
+    margin-top: 2.5em;
+}
+
+.googleimg{
+    aspect-ratio: auto 272 / 92;
+    height: 92px;
+    width: 272px;  
+}
+
+/* .gcolor{
+    color: #4285F4;
+}
+
+.firsto{
+    color: #EA4335;
+}
+
+.secondo{
+    color: #FBBC06;
+}
+
+.lcolor {
+    color: #34A853;
+} */
+
+.userinfo li{
+    cursor:pointer;
+}
+.googlesearch{
```

Comment:
You should make use of linters, another to follow the best coding standard for the language, You can add a stylelint for css

Reply:
- [ ] It will be better to put the link for the student on how can set up linter using config files copied from [config linters repo](https://github.com/microverseinc/linters-config#linters-config). There is a reason why to request change and link for the document by giving a valid reason for desired change that makes code review process seems meaningful.

Reply:
- [ ] It will be better to put the link for the student on how can set up linter using config files copied from [config linters repo](https://github.com/microverseinc/linters-config#linters-config). There is a reason why to request change and link for the document by giving a valid reason for desired change that makes the code review process seems meaningful.
- [ ] I suggest using gifs and emojis if you want, since they are human-friendly 😊